packages/data-point/lib/reducer-list/resolve.test.js

```diff
@@ -0,0 +1,159 @@
+/* eslint-env jest */
```

Comment:
is this new code or should had been a git mv?

Reply:
should have been a git mv